src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public String getRandomBall(Lottery ball) {
+        int number = new Random().nextInt(100);
+        return "Ball has number: " + number + " and color: " + ball.getRandomColor();
```

Comment:
This method should return an instance of a Ball object instead of a String. Create a new Ball object with the generated number and color, and return it.